plugins/atlas/skills/histfitter/references/systematic-types.md

```
@@ -0,0 +1,282 @@
+  factors. Like `overallHistoSys` but the shape is additionally normalized to
+  the control regions. The normalization is factorized out as an `overallSys`.
+- **`shapeSys`**: Bin-by-bin uncorrelated uncertainties. One NP per bin, shared
+  across samples (can merge samples with `syst.mergeSamples(sampleList)`).
+  Constraint can be `"Gaussian"` or `"Poisson"`.
+- **`shapeStat`**: Like `shapeSys` but for MC statistical uncertainty of a
+  single sample. Uses the bin errors from the nominal histogram. Supports a
+  `statErrorThreshold` to ignore bins with small relative errors.
+
+## Choosing a method — decision flowchart
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)